dgisim/src/dices.py

```diff
@@ -279,7 +491,12 @@ class AbstractDices(Dices):
     })
 
     @classmethod
-    def from_pre(cls, omni: int, any: int, element: Optional[Element] = None, num: Optional[int] = None) -> AbstractDices:
+    def from_pre(
+            cls,
+            omni: int,
+            any: int,
+            element: Optional[Element] = None,
+            num: Optional[int] = None) -> AbstractDices:
```

Comment:
I normally format it as

```py
        num: Optional[int] = None
) -> AbstractDices:
```

just like how I format other functions in this project.